packages/postgrest-core/src/lib/get-table-from-url.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Parses a url and returns the table name the url is interacting with.
+ *
+ * For mutations, the .split("?") goes unused.
+ *
+ * @param url The url we are pulling the table name from
+ * @returns Table name
+ */
+export const getTableFromUrl = (url: string): string =>
+  url.split("/").pop()?.split("?").shift() as string;
```

Comment:
This won't work for rpcs. Can you add support for it? Similar to what it proposed here:

https://github.com/psteinroe/supabase-cache-helpers/issues/503#issuecomment-2625681068